### PR TITLE
Target iOS 15 and macOS 12

### PR DESCRIPTION
**Pull request by jacobsimeon**

### CODE REVIEW

**Review by nzeltzer — APPROVED** [review]
lgtm

(I can't speak to the test change, but I trust the test will speak for itself)

File: Shared/Item/ItemListView.swift [hunk 1]
```diff
@@ -23,9 +23,11 @@ struct ItemListView: View {
     var body: some View {
         List(items) { item in
             ZStack(alignment: .leading) {
+                NavigationLink(
+                    destination: ItemDestinationView(item: item)
+                ) { EmptyView() }
+
                 ItemRowView(model: ItemPresenter(item: item))
-                NavigationLink(destination: ItemDestinationView(item: item)) { }
-                    .hidden()
```
Comment (dskuza) [inline Shared/Item/ItemListView.swift:28]: Aha! Nice discovery.

**Review by dskuza — APPROVED** [review]
Looks good to me. I noticed the commit messages aren't the previously used "conventional commit" styling. Should we retro that styling?

**Comment by jacobsimeon** [comment]
> (I can't speak to the test change, but I trust the test will speak for itself)

@nzeltzer looks like iOS 14 could only find that button based on the name of the image icon. iOS 15 does too, unless you've given the button an accessibility identifier, in which case you need to search for it based on the identifier.

**Comment by jacobsimeon** [comment]
@dskuza, I squashed the commits and changed the message to use conventional commits

**Pull request merged**